.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+ignore = E203, E266, E501, W503
+max-line-length = 80
+max-complexity = 18
+select = B,C,E,F,W,T4,B9
```

Comment:
Since we are aligning our coding style with `ethereum/trinity`, could you change the flake8 settings to the ones [here](https://github.com/ethereum/trinity/blob/a6feb6b7364db5c19c9dd74c51df07ad4d3de96d/tox.ini#L13-L16)?

Reply:
Sorry for the misunderstanding. The `flake8` looks good, which complies with the result from black.